### PR TITLE
Nick03022019

### PR DESCRIPTION
**Pull request by NickJacksonDev**
IMPORTANT: 
will probably need to fix the .gitignore file since Ben and I both edited it
may still have problem with the Campaign Manager in installed apps

### CODE REVIEW

**Comment by DelgadoJosh** [comment]
Looks good to me. The html for the campaign manager looks good. I haven't opened up the page on my side, but the code looks good

File: Campaign_Manager/forms.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Campaign
+
+class CampaignForm(forms.ModelForm):
```
Comment (brhiner) [inline Campaign_Manager/forms.py]: Campaign form is a little vague. Maybe CreateCampaignForm?

**Comment by SpinksJared** [comment]
From what I can tell it looks good to me.

File: Campaign_Manager/templates/Campaign_Manager/campaign_manager-home.html [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
     <h1>Campaign Builder!</h1>
+
+    {% for Campaign in campaigns %}
+
+		<h2>{{ Campaign.campaignName }}</h2>
+		<h3>Campaign ID: {{ Campaign.campaignID }}</h3>
+
+    {% endfor %}
+
+    {% for Character in characters %}
```
Comment (brhiner) [inline Campaign_Manager/templates/Campaign_Manager/campaign_manager-home.html]: As discussed offline, we can remove this from this file

**Review by brhiner — CHANGES_REQUESTED** [review]
Code looks good. I've posted a few questions/reminders below to address before you're good to merge
- Don't forget to pull from master and resolve any conflicts before submitting
- Remove DND_Manager/__pycache__/urls.cpython-37.pyc from your commit

**Comment by NickJacksonDev** [comment]
I don't really understand how to edit a merge request once it has been initiated so I am just making a new one with all of Ben's concerns addressed.

**Pull request closed without merging**